criu/pidfd.c

```diff
@@ -49,6 +59,30 @@ int init_dead_pidfd_hash(void)
 	return 0;
 }
 
+static struct dead_count *init_dead_count(int pidfd_owner)
+{
+	struct dead_count *spec_count;
```

Comment:
nit: empty line between code and declarations.

Reply:
no longer needed